vendor/engines/split_accounts/app/models/split.rb

```diff
@@ -0,0 +1,19 @@
+class Split < ActiveRecord::Base
```

Comment:
Part of me wonders if this should have a more explicit name. Either `AccountSplits` or put it under the engine namespace `SplitAccounts::Split`.